pybaseball/playerid_lookup.py

```diff
@@ -47,77 +50,119 @@ def chadwick_register(save: bool = False) -> pd.DataFrame:
+            player_list (list, optional): List of players to seach for (comma delimited). Defaults to None.
+            search (bool, optional): In case of typos, returns players with names close to input. Defaults to False.
+
+        Returns:
+            pd.DataFrame: DataFrame of playerIDs, name, years played
+        """
+
+        # force input strings to lowercase
+        if last:
+            last = last.lower()
+        if first:
+            first = first.lower()
+
+        # if player_list has a value, then the user is passing in a list of players
+        # the list of players may be comma delimited for last, first, or just last
+        if player_list:
+            player_counter = 1
+            for player in player_list:
+                last = player.split(",")[0].strip()
+                first = None
+                if (len(player.split(",")) > 1):
+                    first = player.split(",")[1].strip()
+                if (player_counter == 1):
+                    results = self.search(last, first)
+                else:
+                    results = results.append(self.search(last, first), ignore_index=True)
+                player_counter += 1
+            return results
+
+        if first is None:
+            results = self.table.loc[self.table['name_last'] == last]
+        else:
+            results = self.table.loc[(self.table['name_last'] == last) & (self.table['name_first'] == first)]
+
+        results = results.reset_index(drop=True)
+
+        # If no matches, return 5 closest names
+        if len(results) == 0 and fuzzy:
+            print("No identically matched names found! Returning the 5 most similar names.")
+            similar_names_df = name_similarity(last=last, first=first, player_table=self.table)
+            results = similar_names_df.merge(self.table, left_on=["name_last", "name_first"], right_on=[
+                                             "name_last", "name_first"], how="left")
+
         return results
 
-    if first is None:
-        results = table.loc[table['name_last'] == last]
-    else:
-        results = table.loc[(table['name_last'] == last) & (table['name_first'] == first)]
+    def reverse_lookup(self, player_ids: List[str], key_type=None) -> pd.DataFrame:
+        """Retrieve a table of player information given a list of player ids
 
-    results = results.reset_index(drop=True)
+        :param player_ids: list of player ids
+        :type player_ids: list
+        :param key_type: name of the key type being looked up (one of "mlbam", "retro", "bbref", or "fangraphs")
+        :type key_type: str
 
-    return results
+        :rtype: :class:`pandas.core.frame.DataFrame`
+        """
+        key_types = (
+            'mlbam',
+            'retro',
+            'bbref',
+            'fangraphs',
+        )
 
+        if not key_type:
+            key_type = key_types[0]  # default is "mlbam" if key_type not provided
```

Comment:
Would it be more clear to make "mlbam" the default argument in the function declaration?

Reply:
It would. I migrated existing code pretty thoughtlessly

Reply:
Done, and threw in some F-string updates to there as well